src/openms/include/OpenMS/ANALYSIS/ID/FragmentIndex.h

```diff
@@ -39,6 +39,41 @@ namespace OpenMS
       float precursor_mz;  // charge 1 precursor m/z
     };
 
+    struct sPeptide
+    {
+      UInt32 protein_idx_;
+      UInt32 modification_idx;
+      std::pair<Size, Size> sequence;
```

Comment:
Size can be rater large.
is this just the index in the protein?
then maybe something like
UInt16, UInt16 could safe some space

Reply:
Can I simply cast pair<size_t, size_t> to pair<UInt16, UInt16> ?